app/src/androidTest/java/edu/uwstout/p2pchat/HomeFragmentTest.java

```diff
@@ -96,12 +96,7 @@ public void after()
     public void testDiscoveryList()
     {
         assert(!TestHomeFragment.getNavigated());
-        try {
-            openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getTargetContext());
-        } catch (Exception e) {
-            //This is normal. Maybe we dont have overflow menu.
-        }
```

Comment:
I don't like suppressing an exception, but if the exception is harmless to the test I'll let it slide.